labels.json

```diff
@@ -1,135 +1,316 @@
+  },
+  {
+    "name": "status: needs more information",
+    "description": "An Issue reporting a potential bug that needs more detail in order to help reproduce it",
+    "color": "c6a49a",
+    "aliases": [
+      "status: needs more information",
+      "needs more information",
+      "needs-more-info"
     ]
   },
   {
-    "name": "enhancement",
+    "name": "resolution: can't reproduce",
+    "description": "An Issue reporting a bug that cannot be reproduced",
     "color": "0052cc",
     "aliases": [
-      "enhancement",
-      "feature"
+      "resolution: can't reproduce",
+      "non-reproducible"
     ]
   },
   {
-    "name": "modulesync",
-    "color": "fbca04",
+    "name": "resolution: duplicate",
+    "description": "An Issue raised that has already been reported",
+    "color": "cccccc",
     "aliases": [
-      "msync"
+      "resolution: duplicate",
+      "duplicate"
     ]
   },
   {
-    "name": "needs-docs",
-    "color": "006b75",
+    "name": "resolution: fixed",
+    "description": "An Issue that has been fixed by a Pull Request",
+    "color": "d4c5f9",
     "aliases": [
-      "needs-docs"
+      "resolution: fixed"
     ]
   },
   {
-    "name": "needs-feedback",
-    "color": "5319e7",
+    "name": "resolution: invalid",
+    "description": "An Issue that is determined to not be related to this project",
+    "color": "e6e6e6",
     "aliases": [
-      "feedback",
-      "needs-feedback"
+      "resolution: invalid",
+      "invalid"
     ]
   },
   {
-    "name": "needs-help",
-    "color": "d4c5f9",
+    "name": "resolution: won't fix",
+    "description": "An Issue reporting behaviour that is not intended to be changed in the future",
+    "color": "ffffff",
     "aliases": [
-      "help",
-      "help wanted",
-      "needs-help"
+      "resolution: won't fix",
+      "wont-fix",
+      "wontfix",
+      "wont-fix",
+      "won't-fix"
     ]
   },
   {
-    "name": "needs-rebase",
+    "name": "resolution: works as expected",
+    "description": "An Issue reporting a bug that is actually normal behaviour",
     "color": "207de5",
     "aliases": [
-      "needs-rebase",
-      "rebase"
+      "resolution: works as expected",
+      "not-a-bug"
+    ]
+  },
+  {
+    "name": "keep open",
+    "description": "An Issue or Pull Request that should be kept open",
+    "color": "fbca04",
+    "aliases": [
+      "keep open",
+      "leave-open"
     ]
   },
   {
-    "name": "needs-squash",
+    "name": "for new contributors",
+    "description": "An Issue raising a need for changes, that should be relatively easy to do",
+    "color": "02d7e1",
+    "aliases": [
+      "for new contributors",
+      "beginner",
+      "beginner-friendly",
+      "beginner friendly"
+    ]
+  },
+  {
+    "name": "status: confirmed",
+    "description": "An Issue reporting a bug that is confirmed to affect other users",
     "color": "c2e0c6",
     "aliases": [
-      "needs-squash",
-      "squash"
+      "status: confirmed",
```

Comment:
Not useless, because it helps reduce time spent investigating things which aren't bugs, and helps get people involved in helping narrow down the cause. I wasn't thinking of enforcement, only a way to indicate that further assistance from module users was welcome.